### DISCUSSION TITLE
How to get page from ContentView?

### DISCUSSION THREAD

**Discussion by lanyusan** [dotnet/maui#11360] [Q&A]
I understand custom controls are created with `ContentView` and could be instantiated pretty low in control tree of a page.

To display alerts from `ContentView` it has to be able to get the page instance it is currently in. I haven't found from `ContentView` api doc how to get it. 

Can someone please help?

Thanks.

**Comment by xtuzy** [comment]
you can binding a command of viewmodel, or find page from parents of this CollectionView.

**Comment by XrayHouse** [comment]
Use App.Current.MainPage. for the current page.
